extensions/ui/org.eclipse.smarthome.ui.basic/snippets/group.html

```diff
@@ -2,10 +2,10 @@
 	<span class="mdl-form__icon">
 		<img data-icon="%category%" src="../icon/%category%?state=%state%&format=%icon_type%" />
 	</span>
-	<span class="mdl-form__label">
+	<span %labelstyle% class="mdl-form__label">
 		%label%
 	</span>
-	<div class="mdl-form__text-link-value">%value%</div>
+	<div %valuestyle% class="mdl-form__value mdl-form__text-link-value">%value%</div>
```

Comment:
Please add `font-weight: normal` property in LESS for `.mdl-form__text-link-value`. Otherwise LGTM